awx/main/sortedbulkupdate.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2015 Ansible, Inc.
```

Comment:
I'd like this in a different folder. See `awx/main/utils/mem_inventory.py` as an example, a new file like `awx/main/utils/inventory.py` might be okay, or even adding it to `awx/main/utils/db.py` actually would work fine in my book (because it is a DB utility).

My original thought is that this doesn't have to be specific to the `Host` model. Yeah, you import the `Host` model, but you don't have to. The class can be taken as an argument or obtained indirectly from the type of the objects.